src/provider/provider_os_memory.c

```diff
@@ -191,6 +198,33 @@ static umf_result_t translate_params(umf_os_memory_provider_params_t *in_params,
         return result;
     }
 
+    result =
+        os_translate_memory_flag(in_params->visibility, &provider->visibility);
```

Comment:
please change name to "os_translate_mem_visibility_flag" as this is more consistent with other functions

Reply:
Done